src/lib/CasparCGSocket.ts

```diff
@@ -35,6 +35,7 @@ export class CasparCGSocket extends EventEmitter implements ICasparCGSocket {
 	private _client: net.Socket
 	private _host: string
 	private _port: number
+	private _localAddress: string
```

Comment:
The typings here should be `string | undefined` I think, since that's how you define it in the public interface

Reply:
I just copied the behavior/style of the `host` field, but I think your right: `localAddress` should probably not have a default value from the IPv4 address space. This might break attempts to connect to an IPv6 host. I will have a look at this but need to do some more testing.

Reply:
I fixed it and did some tests with the new default value `undefined`.